### ISSUE TITLE
How to configure the args of examples/speech_recognition/infer.py to reproduce the results in the wav2vec 2.0 paper?

### ISSUE THREAD

**Issue by shiyuzh2007** [pytorch/fairseq#2438]
#### What is your question?
How to configure the args of examples/speech_recognition/infer.py to reproduce the results in the wav2vec 2.0 paper?

#### Code
python examples/speech_recognition/infer.py ./data/librispeech/finetune --task audio_pretraining \
--nbest 1 --path ./models/wav2vec_small_10m.pt --gen-subset "dev_other" --results-path ./decoder_result --w2l-decoder kenlm --lm-model data/librispeech/4-gram.bin --lm-weight 3.23 --word-score -0.26 --sil-weight 0 --criterion ctc --labels ltr --max-tokens 4000000  --post-process letter --lexicon ./libri_lexicon.txt

   

#### What have you tried?
I downloaded the model of wav2vec_small_10m.pt and tried to reproduce the results on dev_other of librispeech. 
I set --lm-weight 3.23 and --word-score -0.26 according to the Table 7 in the paper.
I got the result WER=22.8056% by the above command. But the WER of dev_other is 15.7 in the wav2vec 2.0 paper.
What is wrong with the args? Could you give me a correct args list which can reproduce the results in the paper?
